.github/workflows/client.yml

```diff
@@ -0,0 +1,42 @@
+name: Go
+
+#TODO: extend the conditions once workflow gets tested together with other workflows 
+on:  
+  push:
+    branches:
+      - master
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: satackey/action-docker-layer-caching@v0.0.11
+        continue-on-error: true # ignore the failure of a step and avoid terminating the job
```

Comment:
I'll flag that the comment here mostly seems to be repeating what the flag already communicates (unlike `if: always()` which isn't as obvious).

Reply:
You're right. I'm not changing it though, as this fragment of code will be removed once #2331 gets merged to master.